packages/plugin-user/src/user.js

```diff
@@ -27,6 +28,29 @@ const User = SparkPlugin.extend({
   },
 
   /**
+   * Finishes the signup process for a user that signed up on a mobile device
+   * @param {Object} options
+   * @param {string} options.encryptedQueryString
+   * @returns {Promise<Object>}
+   */
+  activate(options) {
```

Comment:
default params isn't in es2015. Other than the explicit inclusion of decorators, the sdk has a fairly conservative babel config.